app/controllers/api/v1/monsters_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Api::V1::MonstersController < ApplicationController
+  protect_from_forgery unless: -> { request.format.json? }
+
+  def index 
```

Comment:
We'll want an API controller test!

Reply:
We had one but maybe it wasn't uploaded in the PR